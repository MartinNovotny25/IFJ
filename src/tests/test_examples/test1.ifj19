ahoj = 1.2
dublik = 21.3
beta = 'betka je stetka'
