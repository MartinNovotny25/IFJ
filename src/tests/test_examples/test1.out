TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(DOUBLE, 1.200000)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(DOUBLE, 21.300000)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, betka je stetka)
TOKEN(EOL)
TOKEN(EOF)
